### PR TITLE
SISRP-9429 Add API for work experience

### PR DESCRIPTION
**Pull request by ctweney**
https://jira.berkeley.edu/browse/SISRP-9429

### CODE REVIEW

File: fixtures/json/work_experience.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
+{
```
Comment (christianv) [inline fixtures/json/work_experience.json]: @ctweney just as a small fyi, they are still adding a few extra properties to this list.

**Comment by christianv** [comment]
@ctweney lgtm, but would be good for @raydavis, @paulkerschen or @johncrossman to have a look at it as well.

File: spec/models/hub_edos/work_experience_spec.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
```
Comment (paulkerschen) [inline spec/models/hub_edos/work_experience_spec.rb]: @ctweney No biggie but we're still trying to garden these requires out of the specs.

Comment (ctweney) [inline spec/models/hub_edos/work_experience_spec.rb]: ok @paulkerschen  I'll remove. Don't merge yet, I'll push a followup.

Comment (ctweney) [inline spec/models/hub_edos/work_experience_spec.rb]: Actually, scratch that -- I'll open a separate PR. There are a LOT of 'requires' in my code!

**Comment by paulkerschen** [comment]
Looks good to me too. Thanks @ctweney for the comment pre-empting any questions about whether the API response really is structured like that.

**Pull request merged**